httpmw/Ip_cognito_auth_test.go

```diff
@@ -80,27 +99,62 @@ func TestCognitoIpAuth(t *testing.T) {
 			nil,
 		)
 
-	router.Use(IpCognitoAuth(ipRanges, srv))
+	router.Use(IpCognitoAuth(ipRanges, srv, authTestCogntoClientId))
 	router.GET("/login", func(c *gin.Context) {
 		uname, _ := c.Get("username")
 
 		assert.Equal(authTestUsername, *uname.(*string))
 		c.Status(http.StatusOK)
 	})
 	w := httptest.NewRecorder()
-	req, _ := http.NewRequest("GET", "/login", nil)
-	req.Header.Set("Authorization", fmt.Sprintf("%s %s", "Bearer", token))
+	req, err := http.NewRequest("GET", "/login", nil)
+	assert.NoError(err)
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", token))
 	router.ServeHTTP(w, req)
 	assert.Equal(http.StatusOK, w.Code)
 }
 
+func TestCognitoIpAuthTokenFails(t *testing.T) {
+	assert := assert.New(t)
+	ipRanges := authTestIPRanges
```

Comment:
This variable is redundant you can just pass `authTestIPRangesLarge` directly into  `IpCognitoAuth`.